packages/schematics/src/collection/cypress-project/files/tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
This should extend the base `tsconfig` and a lot of settings wouldn't be required